cg_hermes/config/taxprofiler.py

```diff
@@ -0,0 +1,66 @@
+"""Tags that are defined in taxprofiler deliverables mapped to tags used in CG"""
+
+from cg_hermes.config.nextflow import NEXTFLOW_COMMON_TAGS
+
+TAXPROFILER_COMMON_TAGS = {
+    frozenset({"kraken2_combined_report", "kraken2"}): {
+        "is_mandatory": True,
+        "bundle_id": True,
+        "tags": ["kraken2", "kraken2report"],
+        "used_by": ["deliver"],
+    },
+    frozenset({"kraken2_report", "kraken2"}): {
+        "is_mandatory": True,
+        "bundle_id": True,
+        "tags": ["kraken2", "kraken2report"],
+        "used_by": ["deliver"],
+    },
+    frozenset({"krona_kraken_plot", "krona"}): {
+        "is_mandatory": True,
+        "bundle_id": True,
+        "tags": ["kraken2", "krona", "visualization"],
+        "used_by": ["deliver"],
+    },
+    frozenset({"krona_centrifuge_plot", "krona"}): {
+        "is_mandatory": True,
+        "bundle_id": True,
+        "tags": ["centrifuge", "krona", "visualization"],
+        "used_by": ["deliver"],
+    },
+    frozenset({"bracken_combined_report", "bracken"}): {
+        "is_mandatory": True,
+        "bundle_id": True,
+        "tags": ["bracken", "brackenreport"],
+        "used_by": ["deliver"],
+    },
+    frozenset({"bracken_report", "bracken"}): {
+        "is_mandatory": True,
+        "bundle_id": True,
+        "tags": ["bracken", "brackenreport"],
+        "used_by": ["deliver"],
+    },
```

Comment:
Be cautious about using identical tags for different files. We aim to uniquely identify each file so that we can extract them individually in Housekeeper